Sources/tasks/CreatePost-Notify.php

```diff
@@ -76,10 +76,7 @@ public function execute()
 		// Find the people interested in receiving notifications for this topic
 		$request = $smcFunc['db_query']('', '
 			SELECT
-				ln.id_member, ln.id_board, ln.id_topic, ln.sent,
-				mem.email_address, mem.lngfile, mem.pm_ignore_list,
-				mem.id_group, mem.id_post_group, mem.additional_groups,
-				mem.time_format, mem.time_offset, mem.timezone,
+				mem.*, ln.id_board, ln.id_topic, ln.sent,
```

Comment:
to use * is always bad: https://stackoverflow.com/questions/3639861/why-is-select-considered-harmful